src/Mod/PartDesign/Gui/TaskPolarPatternParameters.cpp

```diff
@@ -156,6 +156,7 @@ void TaskPolarPatternParameters::setupUI()
 
     ui->polarAngle->bind(pcPolarPattern->Angle);
     ui->spinOccurrences->setMaximum(INT_MAX);
+    ui->spinOccurrences->setMinimum(2);

```

Comment:
Some months ago the restriction of linear patterns has been lifted to also allow only one occurrence. As far as I can see this wasn't done for polar patterns yet. So, if this makes sense for linear patterns then it also makes perfectly sense for polar patterns.

Reply:
I thought also for linear patterns only one occurrence is senseless, but in contrary to polar patterns, it doesn't lead to an error. So I only changed it for polar patterns since there only 1 occurrence always leads to an error.

Reply:
Allowing one occurrence doesn't harm and in GUI mode you probably won't use it. But the guy who implemented it for the linear pattern had a use case in combination with scripting and expressions.